Milyli.ScriptRunner.ExternalIHelper/Milyli.ScriptRunner.ExternalIHelper.csproj

```diff
@@ -37,9 +37,9 @@
     <DefineConstants Condition="'$(RelativityAPIVersion)' != ''">$(DefineConstants);V$(RelativityAPIVersion.Replace(".", "_"))</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Relativity.Api" Version="$(RelativityVersion)" /> 
-    <PackageReference Include="Relativity.ObjectManager" Version="9.5.162.111" />
-    <PackageReference Include="Relativity.Other" Version="$(RelativityVersion)" />
+    <PackageReference Include="Relativity.Api" Version="9.6.284.6" /> 
```

Comment:
Not sure why these were substituted variables

Reply:
ExternalIHelper is usually pinned to the relativity version because of the changes to the interface, This is why in ExternalIHelper.cs we now have 2 implementations of GetSecretStore